index.html

```diff
@@ -319,30 +277,27 @@ <h1 class="text-sm flex justify-center p-5 dark:bg-ys-black">
                   </button>
                 </div>
                 <div class="p-8 dark:bg-ys-backgroundAndText">
-                  <div id="code-wrapper" data-cy="util:code-editor">
-                    <textarea
-                      class="dark:bg-ys-backgroundAndText bg-ys-amethyst-200 p-5 outline-none"
-                      name=""
-                      id=""
-                      aria-label=""
-                      rows="6"
-                    >
-                    </textarea>
+                  <div>
```

Comment:
I think we need to keep the data-cy attribute for E2E testing with Cypress.
```suggestion
                  <div id="code-wrapper" data-cy="util:code-editor">
```

Reply:
thank you! much appreciated 🥇